### PR TITLE
3.39.1 customer replica

### PR DESCRIPTION
**Pull request by coury-clark**
### Checklist

* [ ] Sister [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph) change:
* [ ] All images have a valid tag and SHA256 sum

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 .github/
 *.md
-rootfs/etc/docker/daemon.json
```
Comment (camdencheek) [inline .prettierignore:3]: I don't think this is supposed to go away

File: prometheus/prometheus_targets.yml [hunk 2]
```diff
@@ -56,7 +56,7 @@
 - labels:
     job: codeinsights-db
   targets:
-    - codeinsights-db:9187
+    - codeinsights-db-exporter:9187
```
Comment (camdencheek) [inline prometheus/prometheus_targets.yml:59]: Just double checking that this got manual eyes

File: pure-docker/deploy.sh [hunk 3]
```diff
@@ -49,6 +48,5 @@ do
 done
 
 for i in $(seq 0 $(($NUM_FRONTEND - 1))); do ./deploy-frontend.sh $i; done
-# Not used in customer-replica branch.
-#./deploy-caddy.sh
+./deploy-caddy.sh
```
Comment (camdencheek) [inline pure-docker/deploy.sh:51]: I think the changes in this file need to not be included as well

**Review by camdencheek — CHANGES_REQUESTED** [review]

File: prometheus/prometheus_targets.yml (hunk 2 above)
Comment (coury-clark) [reply prometheus/prometheus_targets.yml:59]: Yeah I don't really know what this is, but I don't think it's coming from the release tool

File: pure-docker/deploy.sh (hunk 3 above)
Comment (coury-clark) [reply pure-docker/deploy.sh:51]: Ah yikes - thanks!

File: prometheus/prometheus_targets.yml (hunk 2 above)
Comment (camdencheek) [reply prometheus/prometheus_targets.yml:59]: Cool. Seems to be in the right commit either way.

**Pull request closed without merging**